From 9d88937bec37e7d166764614164d503aea60d04c Mon Sep 17 00:00:00 2001
From: Irina Tirdea <irina.tirdea@intel.com>
Date: Tue, 26 Jun 2012 18:53:01 +0300
Subject: Fix PTHREAD_STACK_MIN definition in pthread.h

The definition of PTHREAD_STACK_MIN in pthread.h
depends on PAGE_SIZE. PAGE_SIZE is not defined
in pthread.h, so whoever wants to use PTHREAD_STACK_MIN
needs to include first asm/page.h.

Including pthread.h should be enough to have a valid
definition of PTHREAD_STACK_MIN. Including asm/page.h
in pthread.h so that PAGE_SIZE is visibile and the
definition for PTHREAD_STACK_MIN valid.

For: AIA-452
Change-Id: I2a0de73aa6482894b03ed792cb69485ec61a217e
Signed-off-by: Irina Tirdea <irina.tirdea@intel.com>
---
 libc/include/pthread.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/libc/include/pthread.h b/libc/include/pthread.h
index dbdee70..3b6068f 100644
--- a/libc/include/pthread.h
+++ b/libc/include/pthread.h
@@ -33,6 +33,7 @@
 #include <sched.h>
 #include <limits.h>
 #include <sys/types.h>
+#include <asm/page.h>
 
 /*
  * Types
-- 
1.7.4.1

